modules/earthengine-layers/src/earth-engine-layer.js

```diff
@@ -113,9 +112,7 @@ export default class EarthEngineLayer extends CompositeLayer {
         id: this._getLayerId(getTileUrl),
         async getTileData({x, y, z}) {
           const imageUrl = getTileUrl(x, y, z);
-          const image = await loadImageBitmap(imageUrl);
-          // const imageData = await getImageData(image);
-
+          const image = await load(imageUrl, ImageLoader);
```

Comment:
The fast `ImageBitmap` loading path is still not the default, we need to provide an option to enable it. You need to check the docs but think it is `load(imageUrl, ImageLoader, {image: {type: 'imagebitmap'}})`.

Reply:
Will setting `type` still handle fallbacks correctly?

Reply:
No as indicated above, I don't think it will right now.